guides/common/modules/snip_common-compute-resource-prereqs.adoc

```diff
@@ -1,16 +1,25 @@
 ifdef::foreman-el,katello[]
-* The installation media for the operating systems that you want to use to provision hosts.
+* Provide the installation medium for the operating systems that you want to use to provision hosts.
 If the Katello plug-in is installed, you can use synchronized content repositories for {RHEL}.
 For more information, see {ContentManagementDocURL}Synchronizing_Repositories_content-management[Syncing Repositories] in _{ContentManagementDocTitle}_.
-* If the Katello plug-in is installed, an activation key for host registration.
+* If the Katello plug-in is installed, provide an activation key for host registration.
 For more information, see {ContentManagementDocURL}Creating_an_Activation_Key_content-management[Creating An Activation Key] in _{ContentManagementDocTitle}_.
 endif::[]
 ifdef::foreman-deb[]
-  * The installation media for the operating systems that you want to use to provision hosts.
+* Provide the installation medium for the operating systems that you want to use to provision hosts.
 endif::[]
 ifdef::satellite[]
-  * Synchronized content repositories for {RHEL}.
+* You can use synchronized content repositories for {RHEL}.
 For more information, see {ContentManagementDocURL}Synchronizing_Repositories_content-management[Syncing Repositories] in _{ContentManagementDocTitle}_.
-  * An activation key for host registration.
+* Provide an activation key for host registration.
+For more information, see {ContentManagementDocURL}Creating_an_Activation_Key_content-management[Creating An Activation Key] in _{ContentManagementDocTitle}_.
+endif::[]
+ifdef::orcharhino[]
+* Provide the installation medium for the operating systems that you want to use to provision hosts.
+ifdef::red_hat_enterprise_linux[]
```

Comment:
```suggestion
ifdef::red_hat_enterprise_linux[]
```
I guess this build is new. Never came across it before.

Reply:
Ah yes, it's an attribute we're currently using downstream only. In upstream, each build/guide (incl. orcharhino) should just evaluate to "false" and not show the content.

Reply:
Alright. Rest looks good to me 👍

Reply:
Bit late to the party, but such a thing is good to mention somewhere, even if it's just a commit message. Otherwise people like me might try to optimize it in the future and break something you rely on.

Reply:
I will bring more changes like this upstream & explain what exactly is going on!